recipe/run_test.py

```diff
@@ -83,7 +80,6 @@
 import strop
 import time
 import test
-import test.test_support
```

Comment:
Why was this test dropped?

Reply:
Turns out some content got shuffled around in Python 2.7.14. Fixing in PR ( https://github.com/conda-forge/python-feedstock/pull/168 ).